### PR TITLE
Setup placeholder pages, components, auth & api endpoints

### PR DESCRIPTION
**Pull request by dride-for-sure**

### CODE REVIEW

**Review by fabianschmauder — COMMENTED** [review]
LGTM ;) a few comments

File: frontend/src/contexts/AuthProvider.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const AuthProvider = ({ children }) => {
+  const [token, setToken] = useState('');
+
+  const isTokenExpired = (tokenToTest) => {
+    try {
+      const decodedToken = decode(tokenToTest);
+      if (decodedToken.exp < Date.now() / 1000) {
+        return true;
+      }
+      return false;
```
[the remark this answers is not in the document]
Comment (dride-for-sure) [reply frontend/src/contexts/AuthProvider.js:17]: Well...yes :(

File: frontend/src/containers/cms/school/SchoolList.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import SideBar from '../../../components/SideBar';
+import Center from '../../../components/structure/Center';
+import CmsGrid from '../../../components/structure/CmsGrid';
+import CmsWrapper from '../../../components/structure/CmsWrapper';
+import MaxWidth from '../../../components/structure/MaxWidth';
+
+export default function SchoolList() {
+  return (
+    <CmsWrapper>
```
[the remark this answers is not in the document]
Comment (dride-for-sure) [reply frontend/src/containers/cms/school/SchoolList.js:9]: The `CmsWrapper` consists of Logo + Navigation ... am i missing something?

**Comment by dride-for-sure** [comment]
Thanks for the review @fabianschmauder !

**Pull request merged**